themes/aframe/source/releases/latest/aframe.js

```diff
@@ -60277,3 +60277,4 @@ module.exports={
 },{}]},{},[28])(28)
 });
 //# sourceMappingURL=aframe.js.map
+console.warn('The URL `https://aframe.io/releases/latest/aframe.js` is deprecated and is permanently fixed to v0.1.3. It is recommended to either upgrade or fix to a specific version like `https://aframe.io/releases/0.1.3/aframe.js`');
```

Comment:
can we link to the Downloads/Builds URLs? https://aframe.io/docs/guide/getting-started.html#Builds - if I read this, I would probably just use `0.1.3` and not upgrade at this time.